ToolsAndDebugging.md

```diff
@@ -2,6 +2,11 @@
 
 For detailed instructions on how to get started with contributing to _fastlane_, first check out [YourFirstPR.md][first-pr] and [Testing.md](Testing.md). This guide will focus on more advanced instructions on how to debug _fastlane_ and _spaceship_ issues and work on patches.
 
+## Experiment with the Fastlane internals
+
+Open a _fastlane_ console by running `fastlane console` inside or outside your project.
+This will allow you to invoke any of the fastlane modules and classes and test their behavior.
```

Comment:
Either

```suggestion

This will allow you to invoke any of the fastlane modules and classes and test their behavior.
```

or both lines in 1 line, because the way it is currently, that's how they will be presented, and without a space in-between them, like:

`…outside your project.This will allow you…`

Reply:
Also:
```suggestion
This will allow you to invoke any of the _fastlane_ modules and classes and test their behavior.
```